.github/workflows/bundle-size.yml

```diff
@@ -113,8 +113,7 @@ jobs:
           npm run quick-build
 
           cd ../bundle-size
-          rm -rf node_modules/@cloudscape-design/components
-          cp -r $GITHUB_WORKSPACE/components/lib/components node_modules/@cloudscape-design/components
+          npm i $GITHUB_WORKSPACE/components/lib/components
```

Comment:
Copying components lib to node_modules does not work when dependencies were added/updated. Instead, the npm install from folder does the trick.